logrus.go

```diff
@@ -176,6 +176,12 @@ type FieldLogger interface {
 	// IsPanicEnabled() bool
 }
 
+type Ext2FieldLogger interface {
```

Comment:
I'd rather not have this new interface.
If a package user needs that, they can easily define it at use side